judge/comments.py

```diff
@@ -72,6 +72,16 @@ def post(self, request, *args, **kwargs):
         if self.is_comment_locked():
             return HttpResponseForbidden()
 
+        parent = request.POST.get('parent')
+        if parent:
+            try:
+                parent = Comment.objects.filter(hidden=False).get(id=int(parent))
```

Comment:
Why are you doing any of that? Just do something like: `Comment.objects.filter(hidden=False, id=int(parent), page=page).exists()`, and freak out when it's `False`. Also remember to catch `ValueError` from `int(parent)`.